defi/src/protocols/data3.ts

```diff
@@ -24511,21 +24511,25 @@ const data3: Protocol[] = [
     address: null,
     symbol: "-",
     url: "https://amet.finance",
-    description: "Amet Finance is a decentralized finance platform for on-chain bonds, providing secure financial transactions and investment opportunities in the blockchain ecosystem",
-    chain: "Manta",
+    description: "Amet Finance is a decentralized platform offering on-chain bond issuance and management services. It enables users to issue, trade, and manage bonds directly on the blockchain, providing a transparent, secure, and efficient environment for both issuers and investors. Amet Finance emphasizes user autonomy, allowing for customized bond terms and leveraging blockchain technology to democratize access to financial instruments.",
+    chain: "Base",
     logo: `${baseIconsUrl}/amet-finance.jpg`,
-    audits: "2",
+    audits: "4",
     audit_note: null,
     gecko_id: null,
     cmcId: null,
-    category: "Services",
-    chains: ["Manta"],
+    category: "Lending",
```

Comment:
Why change the category?

Reply:
At first we selected it wrong, but I switched back to Services, so you won't have UI problems